docs/how-to-contribute/index.md

```diff
@@ -12,15 +12,29 @@ also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
 
 ## Build
 
-Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory. Use the `--prod` flag for a production build.
+Run `npm run build` to build the app. The build artifacts will be stored in the `dist/` directory. Run the command `npm run build-prod` for a production build.
 
-## Running unit tests
+## Test
 
-Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.io).
+The following tests (unit, e2e and lint) are part of the Github Actions (CI) workflow and has to be run successfully before code can be merged into main branch.
```

Comment:
suggestion: before THE code can be merged into THE main branch.

Reply:
Done in 8cb387c